### DISCUSSION TITLE
lanraragi creating file“temp-rewriteaof-xxxx.aof”every second,in redis logs: rename appendonly.aof error: Permission denied

### DISCUSSION THREAD

**Discussion by ztc8069** [Difegue/LANraragi#701] [Q&A] [answered]
in windows or docker,i find when run a lanraragi,it will write harddisk unstoppable.
in database,it creating file“temp-rewriteaof-xxxx.aof”every second,appendonly.aof awlays 0kb
redis logs：Error trying to rename the temporary AOF file temp-rewriteaof-bg-4820.aof into appendonly.aof: Permission denied
when i delete appendonly.aof (0kb) ,programs will operating normally（appendonly.aof saved）,but when i restarts lanraragi,the  problem occurs again！
![GIF 2022-9-5 16-33-59](https://user-images.githubusercontent.com/61111781/188405705-2aaa4852-4cac-4a2e-bdf7-56490c24ecf6.gif)

**Comment by Difegue** [comment]
Sounds like something is messing with the permissions of `appendonly.aof` after it's created -- Maybe something in your Docker binding setup, or an external Windows program?  

It's strange that it doesn't seem to affect the `database.rdb` backup however 🤔

**Reply by Difegue** [reply]
Converted to a discussion since this likely isn't a LRR bug - lmk if you still have the issue!

**Reply by ztc8069** [reply]
I found reason. my hdd used REFS filesystem，i formated my hdd to NTFS filesystem and everything is ok

**Comment by Difegue** [comment, marked as answer]
as per the comment above, the issue came from using a different filesystem.
> I found reason. my hdd used REFS filesystem，i formated my hdd to NTFS filesystem and everything is ok
